packages/nestjs-module/src/ogma.service.ts

```diff
@@ -23,91 +25,118 @@ export class OgmaService implements LoggerService {
   constructor(
     @InjectOgma() ogma?: Ogma,
     @Optional() @InjectOgmaContext() context?: string,
+    @Optional() private readonly requestContext?: RequestContextHost,
   ) {
     this.context = context || '';
     this.ogma = ogma ?? new Ogma();
   }
 
+  private getRequestId(
```

Comment:
Does this handle the cases for GQL or WS?

Reply:
Honestly I checked only GRPC and HTTP but GRPC should works on Nest just like GQL and WS. But yeah, that is thing to test before it will be merged and published